hooks/afterPluginAdd.js

```diff
@@ -1,4 +1,5 @@
 module.exports = function (ctx) {
+
     if (ctx.cmdLine.includes("cordova-plugin-code-push")) {
```

Comment:
Please replace this `includes()` method with `indexOf()`
Let's get rid of all `includes()` here for backword compatibility